torch/csrc/jit/codegen/cuda/lower_loops.cpp

```diff
@@ -178,20 +178,33 @@ void LoopNestGenerator::initReduction(
     kir::Expr* alloc_expr) {
   const auto gpu_lower = GpuLower::current();
 
+  // This is a workaround to handle size-1 reduction, i.e. squeeze ops,
+  // and will be removed once we structurally refactor the way we handle
+  // such reductions, i.e. convert them to SET etc.
```

Comment:
Out of curiosity, what happens today if we tried to convert a trivial reduction op to a set? I assume it will break root mapping? CC @naoyam